draft-irtf-cfrg-frost.md

```diff
@@ -996,14 +996,14 @@ and 3) keep secret values confidential.
   Inputs:
   - s, a group secret, `Scalar` in `GF(p)`, that MUST be derived from at least `Ns` bytes of entropy
   - n, the number of shares to generate, an integer
-  - t, the threshold of the secret sharing scheme, an integer
+  - THRESHOLD_LIMIT, the threshold of the secret sharing scheme, an integer
 
   Outputs:
   - signer_private_keys, `n` shares of the secret key `s`, each a `Scalar` value in `GF(p)`.
   - vss_commitment, a vector commitment of `Element`s in `G`, to each of the coefficients in the polynomial defined by secret_key_shares and whose constant term is s.
 
-  def trusted_dealer_keygen(s, n, t):
-    signer_private_keys, coefficients = secret_share_shard(secret_key, n, t)
+  def trusted_dealer_keygen(s, n, THRESHOLD_LIMIT):
```

Comment:
Should we also use `MAX_SIGNERS` instead of `n` everywhere?

Reply:
Sure, good idea.